src/EA.Weee.RequestHandlers/AatfReturn/ObligatedSentOn/SentOnAatfSiteDataAccess.cs

```diff
@@ -30,7 +30,12 @@ public Task UpdateWithOperatorAddress(WeeeSentOn weeeSentOn, AatfAddress @operat
             return context.SaveChangesAsync();
         }
 
-        public async Task<AatfAddress> GetWeeeSentOnAddress(Guid id)
+        public async Task<AatfAddress> GetWeeeSentOnOperatorAddress(Guid id)
+        {
```

Comment:
Can we create some tests similar to ReturnSchemeDataAccessTests to GetWeeeSentOnOperatorAddress  & GetWeeeSentOnSiteAddress